corehq/apps/app_manager/views/form_designer.py

```diff
@@ -76,6 +76,19 @@ def form_designer(request, domain, app_id, module_id=None, form_id=None,
         'custom_intents': domain_has_privilege(domain, privileges.CUSTOM_INTENTS),
     })
 
+    scheduler_data_nodes = []
+    if getattr(module, 'has_schedule', False):
+        scheduler_data_nodes = [
+            "unscheduled_visit",
+            "current_visit_number",
+            "next_visit",
+            "next_visit_date",
+            "next_due",
+        ]
```

Comment:
@proteusvacuum I changed over to use the consts. I noticed `next_visit` isn't in there (and I couldn't find it elsewhere). Is the confluence page out of date?